DEVELOP.md

```diff
@@ -30,6 +27,22 @@ python3 -m pip install --user --upgrade twine
 python3 -m twine upload dist/*
 ```
 
+### R
+
+From the command prompt, navigate to `src/r`. Open R and run:
+
+```r
+devtools::document()
+devtools::install()
+unlink("man")
```

Comment:
what is the purpose of this?

Reply:
This is basically what you wrote in `build_package.sh`. Generate package documentation, install the package, and remove the `man` folder created by `devtools::document()`.

Reply:
I understand but why do you need to remove it as it's not tracked by git (i.e. it's in gitignore)

Reply:
Oh I did not notice that `man` is git ignored. Then it make sense to remove that line.